build/README.md

```diff
@@ -0,0 +1,24 @@
+# Convox Build Action
+This Action [builds](https://docs.convox.com/deployment/builds) an app based on a [convox.yml](https://docs.convox.com/application/convox-yml) so that app can be deployed on Convox
+
+## Inputs
+### `password`
+**Required** The value of your [Convox Deploy Key](https://docs.convox.com/console/deploy-keys)
```

Comment:
Value strikes me as a bit strange here.. I don't think you would say "The value of your password"